.circleci/config.yml

```diff
@@ -7,10 +7,10 @@ _setup_environment:
     # Following tutorial on https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu
     name: Install Gcloud
     command: |
```

Comment:
The previous method does not install the latest gcloud. This causes the test to fail.